python/cuxfilter/charts/core/core_chart.py

```diff
@@ -24,7 +24,7 @@ class BaseChart:
     data_points: int = 0
     filter_widget = None
     _library_specific_params: Dict[str, str] = {}
-    _stride = None
+    stride = None
```

Comment:
Removed a property from core_charts `self._stride` getter and setter, which was redundant (previously had some logic which was later moved to `compute_stride` fn